exporter/clickhouselogsexporter/migrations/000001_init_db.up.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE IF NOT EXISTS logs (
+	timestamp UInt64 CODEC(Delta, ZSTD(1)),
```

Comment:
So when I get the timestamp in the exporter it's of the type `Timestamp` so regardless if I change it to int64 in my exporter/DB I will get uint64.

Reply:
I don't have strong opinion on this. I would be inclined to store timestamps as signed numbers mostly because be ClickHouse `DateTime64` or time module they are int64 but I am fine without also if it requires lot of work to convert it.